sympy/core/relational.py

```diff
@@ -1066,9 +976,12 @@ class GreaterThan(_Greater):
     rel_op = '>='
 
     @classmethod
-    def _eval_relation(cls, lhs, rhs):
-        # We don't use the op symbol here: workaround issue #7951
-        return _sympify(lhs.__ge__(rhs))
+    def _eval_fuzzy_relation(cls, lhs, rhs):
+        val = is_ge(lhs, rhs)
+        if val is None:
```

Comment:
There are a few of these `_eval_fuzzy_relation` methods and they differ only in which function they call (here `is_ge` for evaluating). Perhaps it would make more sense to move the other logic to the superclass `_eval_relation` method keeping only the call to `is_ge` here since that's the only bit that it unique to this class...

Reply:
Yes, I agree that would be quite a bite better